### PR TITLE
Adding TestVectors to consensus-specs.

### LINKED ISSUE #25: Cryptol to Python test bench
We want to test out the following Cryptol functions against the corresponding Deneb Python Spec implementations:
- [x] [evaluate_polynomial_in_evaluation_form](https://github.com/GaloisInc/ckzg-eip-4844-verification/issues/new#evaluate_polynomial_in_evaluation_form): large inputs but all fixed sized values (`BlsPolynomial` is 4096 256-bit values)
- [x] [compute_quotient_eval_within_domain](https://github.com/GaloisInc/ckzg-eip-4844-verification/issues/new#compute_quotient_eval_within_domain): large inputs but all fixed sized values (`BlsPolynomial` is 4096 256-bit values)
- [x] [compute_kzg_proof_impl](https://github.com/GaloisInc/ckzg-eip-4844-verification/issues/new#compute_kzg_proof_impl):  arge inputs but all fixed sized values (`BlsPolynomial` is 4096 256-bit values)
- [x] [compute_kzg_proof](https://github.com/GaloisInc/ckzg-eip-4844-verification/issues/new#compute_kzg_proof) arge inputs but all fixed sized values (`BlsBlob` is the same size as the `BlsPolynomial`)
- [x] [verify_kzg_proof_impl](https://github.com/GaloisInc/ckzg-eip-4844-verification/issues/new#verify_kzg_proof_impl): for this one we will need to create a generator for G1 points
- [x] [verify_kzg_proof](https://github.com/GaloisInc/ckzg-eip-4844-verification/issues/new#verify_kzg_proof): for this one we will need to create a generator for G1 points that then get compressed

### PR DESCRIPTION
**Pull request by mccleeary-galois**
Close #25

### CODE REVIEW

File: spec/TestVectors/add-tests-to-consensus-specs.patch
Comment (mccleeary-galois) [inline spec/TestVectors/add-tests-to-consensus-specs.patch:1]: Note that the bulk of the work is this patch to `consensus-specs` which adds these test vectors showing. (File is large due to the large test vectors.

Comment (b13decker) [reply spec/TestVectors/add-tests-to-consensus-specs.patch:1]: Excellent!

**Comment by mccleeary-galois** [comment]
> Do we need the `z_bytes_generator` function, since it isn't doing any special handling?

Probably not.

**Comment by b13decker** [comment]
> > Do we need the `z_bytes_generator` function, since it isn't doing any special handling?
> 
> Probably not.

But since we need some generator for type `Bytes32`, let's just keep this one.

**Review by b13decker — APPROVED** [review]

**Pull request merged**